### ISSUE TITLE
about DeepFilterNet2 voicebank+demand test set results

### ISSUE THREAD

**Issue by IMLHF** [Rikorose/DeepFilterNet#291]
In the paper DeepFilterNet2, whether the model tested using VoiceBank+demand test set is trained using VoiceBank+demand training set. If not, wouldn't comparisons with other papers be meaningless because the training data are completely different.
